gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-core/src/main/java/io/gravitee/am/gateway/handler/root/resources/endpoint/webauthn/WebAuthnRegisterEndpoint.java

```diff
@@ -113,7 +119,9 @@ private void renderPage(RoutingContext routingContext) {
 
             final String action = UriBuilderRequest.resolveProxyRequest(routingContext.request(), routingContext.request().path(), queryParams, true);
             final String skipAction = UriBuilderRequest.resolveProxyRequest(routingContext.request(), routingContext.request().path(), queryParams.set("skipWebAuthN", "true"), true);
-
+            if(isEnrollingFido2Factor(routingContext)){
+                routingContext.put(ConstantKeys.MFA_ENROLLING_FIDO2_FACTOR, "true");
```

Comment:
Looks like this entry is never used, if it is true, we can remove the constant declaration here https://github.com/gravitee-io/gravitee-access-management/pull/1869/files#diff-07f22283a8fccd224da72f43b10e63a16a5f1cde5a716172f924a40ccc0808aaR79

Reply:
this key is used in a html file.